src/render/actor_renderer.cpp

```diff
@@ -111,9 +111,10 @@ bool getNextTextId( RenderableSprites_t::const_iterator& i, RenderableSprites_t:
     }
     TexId = i->Spr->TexId;
     (*Positions++) = glm::vec2( i->PositionC->GetX(), i->PositionC->GetY() ) + i->RelativePosition;
-    (*Headings++) = ( GLfloat )i->PositionC->GetOrientation();
+    (*Headings++) = ( GLfloat )i->PositionC->GetOrientation() + i->RelativeOrientation;
 
-    float const radius = ( i->CollisionC != nullptr ? i->CollisionC->GetRadius() : 50 )*i->Anim->GetScale();
+    float const radius = (( i->CollisionC != nullptr ? i->CollisionC->GetRadius() : 50 )
+                            + i->RelativeRadius) *i->Anim->GetScale();
```

Comment:
what's the max. of this? is it possible that isvisible returns false, but the actor is actually visible with the relativeradius addition?